readme.md

```diff
@@ -216,6 +212,21 @@ node_labels:
   - nais.io/storage-nodei=true
   - nais.io/role=worker
 
+[coreos]
+nodename.domain.com
+nodename.domain.com
+
+# file: group_vars/storage_nodes
```

Comment:
Hvorfor ta med denne kommentaren?

Reply:
Bare for å vise at man må spesifisere coreos noder i egen seksjon.